resources/ingress.yaml

```diff
@@ -1,23 +0,0 @@
-apiVersion: extensions/v1beta1
```

Comment:
Do we not want to keep the ingress configuration as an example in this repo too? https://github.com/openshift/continuous-release-jobs/blob/master/config/clusters/dpcr/services/sippy/ingress-sippy.yaml

Reply:
I wanted the directory to be usable with a straight kubectl apply -f dir/, and the ingress felt too deployment specific. I don't think devs or e2e needs it, we port forward, so thought it would be best to leave it out and cite it in the README as an implementation detail of whomever is running sippy.  

Is there a way to have a generic Ingress setup considering the TLS/domains/hosts in that example linked above?